server/services/datacommons.py

```diff
@@ -172,24 +172,21 @@ def obs_series_within(parent_entity, child_type, variables):
       })
 
 
-def v2observation(select, entity, variable):
+def v2observation(select, entities, variables):
   """
   Args:
     select: A list of select props.
-    entity: A dict in the form of {'dcids':, 'expression':}
-    variable: A dict in the form of {'dcids':, 'expression':}
+    entities: A list of entities.
+    variables: A list of variables.
 
   """
-  if 'dcids' in entity:
-    entity['dcids'].sort()
```

Comment:
if mutating is an issue, can do entity['dcids'] = sorted(entity['dcids'])

Reply:
that still mutates entity?  see if this works.